tests/testscript.py

```diff
@@ -1246,10 +1246,66 @@ def test_live_migration_with_vcpu_pinning(self):
         assert int(taskset_vcpu0_controller, 16) == int(taskset_vcpu0_compute, 16)
         assert int(taskset_vcpu2_controller, 16) == int(taskset_vcpu2_compute, 16)
 
+    def test_bdf_assignment(self):
+        """
+        Test if all BDFs are correctly assigned in different scenarios:
+            1) After VM creation and hotplugging a device
+            2) If a devices is removed and it's BDF reused
+            3) Two devices that use the same BDF (expected to fail)
+            4) After live migration (all devices retain assigned BDFs)
+
+        In the current implementation, CHV and libvirt will instantiate
```

Comment:
I think this comment belongs to the constants below and not the function - otherwise, this comment will be heavily outdated in 3-6 months.